README.md

```diff
@@ -1 +1,11 @@
-# viam-typescript-sdk
\ No newline at end of file
+# Viam TypeScript SDK
+
+## (In)stability Notice
+
+> **Warning**
+> This is a beta release of the Viam TypeScript SDK. Stability is not guaranteed. Breaking changes are likely to occur, and occur often.
```

Comment:
let's not say beta here at first. this is definitely alpha at this point

Reply:
```suggestion
> This is an alpha release of the Viam TypeScript SDK. Stability is not guaranteed. Breaking changes are likely to occur, and occur often.
```